### ISSUE TITLE
I can not install in debian 9.2

### ISSUE THREAD

**Issue by arodu** [cedl38/xfce4-windowck-plugin#44]
```
make
make  all-recursive
make[1]: se entra en el directorio '/home/arodu/Descargas/xfce4-windowck-plugin'
Making all in common
make[2]: se entra en el directorio '/home/arodu/Descargas/xfce4-windowck-plugin/common'
  CC       libwck_common_la-wck-plugin.lo
In file included from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-plugin-provider.h:23:0,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-macros-46.h:26,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-plugin.h:28,
                 from wck-plugin.h:25,
                 from wck-plugin.c:29:
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/libxfce4panel.h:33:38: fatal error: libxfce4panel/xfce-hvbox.h: No existe el fichero o el directorio
 #include <libxfce4panel/xfce-hvbox.h>
                                      ^
compilation terminated.
Makefile:515: fallo en las instrucciones para el objetivo 'libwck_common_la-wck-plugin.lo'
make[2]: *** [libwck_common_la-wck-plugin.lo] Error 1
make[2]: se sale del directorio '/home/arodu/Descargas/xfce4-windowck-plugin/common'
Makefile:479: fallo en las instrucciones para el objetivo 'all-recursive'
make[1]: *** [all-recursive] Error 1
make[1]: se sale del directorio '/home/arodu/Descargas/xfce4-windowck-plugin'
Makefile:409: fallo en las instrucciones para el objetivo 'all'
make: *** [all] Error 2
```

**Comment by phw** [comment]
Same here. Also tried changing to `libxfce4panel-1.0` as suggested for Ubuntu 17.04 in https://github.com/cedl38/xfce4-windowck-plugin/issues/39, but this didn't work for Debian Stretch.

**Comment by zlamalp** [comment]
Hi, @arodu and @phw you must download ``libxfce4panel/xfce-hvbox.h`` source file and store it to expected location. It can be found on github somewhere. I believe this was removed from xfce-panel sources because of GTK2->GTK3 migration. But once build GTK2 panel plugin works without problems.

**Comment by phw** [comment]
Yes, I already had it working using this workaround :)

So the proper solution is migrating this plugin to GTK3?

**Comment by zlamalp** [comment]
@phw yes, plugin needs to be migrated to GTK3.

**Comment by pablogzlezmora** [comment]
@zlamalp, I have download the file https://github.com/raboof/xfce4-panel/blob/master/libxfce4panel/xfce-hvbox.h to /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel. With "make" command I got this output (some sentences in Spanish).

What mistake I have made?

Thanks.

```
root@pablo:/home/pablo/Descargas/wd/xfce4-windowck-plugin-master# make
(CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/bash /home/pablo/Descargas/wd/xfce4-windowck-plugin-master/missing autoheader)
rm -f stamp-h1
touch config.h.in
cd . && /bin/bash ./config.status config.h
config.status: creating config.h
config.status: config.h is unchanged
make  all-recursive
make[1]: se entra en el directorio '/home/pablo/Descargas/wd/xfce4-windowck-plugin-master'
Making all in common
make[2]: se entra en el directorio '/home/pablo/Descargas/wd/xfce4-windowck-plugin-master/common'
  CC       libwck_common_la-wck-plugin.lo
In file included from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/libxfce4panel.h:33:0,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-plugin-provider.h:23,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-macros-46.h:26,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-plugin.h:28,
                 from wck-plugin.h:25,
                 from wck-plugin.c:29:
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:7:1: error: expected identifier or ‘(’ before ‘<’ token
 <!DOCTYPE html>
 ^
In file included from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/libxfce4panel.h:33:0,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-plugin-provider.h:23,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-macros-46.h:26,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-plugin.h:28,
                 from wck-plugin.h:25,
                 from wck-plugin.c:29:
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:30:45: error: stray ‘\302’ in program
   <title>xfce4-panel/xfce-hvbox.h at master �� raboof/xfce4-panel · GitHub</title>
                                             ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:30:46: error: stray ‘\267’ in program
   <title>xfce4-panel/xfce-hvbox.h at master �� raboof/xfce4-panel · GitHub</title>
                                              ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:30:67: error: stray ‘\302’ in program
   <title>xfce4-panel/xfce-hvbox.h at master · raboof/xfce4-panel �� GitHub</title>
                                                                   ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:30:68: error: stray ‘\267’ in program
   <title>xfce4-panel/xfce-hvbox.h at master · raboof/xfce4-panel �� GitHub</title>
                                                                    ^
In file included from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/libxfce4panel.h:33:0,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-plugin-provider.h:23,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-macros-46.h:26,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-plugin.h:28,
                 from wck-plugin.h:25,
                 from wck-plugin.c:29:
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:159:8: warning: missing terminating ' character
   <!-- '"` --><!-- </textarea></xmp> --></option></form><form class="js-site-search-form" data-scoped-search-url="/raboof/xfce4-panel/search" data-unscoped-search-url="/search" action="/raboof/xfce4-panel/search" accept-charset="UTF-8" method="get"><input name="utf8" type="hidden" value="&#x2713;" />
        ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:159:8: error: missing terminating ' character
   <!-- '"` --><!-- </textarea></xmp> --></option></form><form class="js-site-search-form" data-scoped-search-url="/raboof/xfce4-panel/search" data-unscoped-search-url="/search" action="/raboof/xfce4-panel/search" accept-charset="UTF-8" method="get"><input name="utf8" type="hidden" value="&#x2713;" />
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/libxfce4panel.h:33:0,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-plugin-provider.h:23,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-macros-46.h:26,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-plugin.h:28,
                 from wck-plugin.h:25,
                 from wck-plugin.c:29:
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:321:48: error: invalid suffix "a780899cf33b9ba055f60b73bd910af" on integer constant
   <!-- blob contrib key: blob_contributors:v21:6a780899cf33b9ba055f60b73bd910af -->
                                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:424:5: error: expected identifier or ‘(’ before ‘<’ token
     </div>
     ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:576:70: error: stray ‘#’ in program
        <td id="LC28" class="blob-code blob-code-inner js-file-line">#<span class="pl-k">ifndef</span> __XFCE_HVBOX_H__</td>
                                                                     ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:580:70: error: stray ‘#’ in program
        <td id="LC29" class="blob-code blob-code-inner js-file-line">#<span class="pl-k">define</span> <span class="pl-en">__XFCE_HVBOX_H__</span></td>
                                                                     ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:589:70: error: stray ‘#’ in program
        <td id="LC31" class="blob-code blob-code-inner js-file-line">#<span class="pl-k">include</span> <span class="pl-s"><span class="pl-pds">&lt;</span>gtk/gtk.h<span class="pl-pds">&gt;</span></span></td>
                                                                     ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:589:149: error: expected identifier or ‘(’ before ‘<’ token
 s="pl-k">include</span> <span class="pl-s"><span class="pl-pds">&lt;</span>gtk/gtk.h<span class="pl-pds">&gt;</span></span></td>
                                                                     ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:589:190: error: expected identifier or ‘(’ before ‘<’ token
 "><span class="pl-pds">&lt;</span>gtk/gtk.h<span class="pl-pds">&gt;</span></span></td>
                                                                     ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:607:168: error: expected identifier or ‘(’ before ‘<’ token
 ef</span> <span class="pl-k">struct</span> _XfceHVBoxClass XfceHVBoxClass;</td>
                                                                           ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:611:163: error: expected identifier or ‘(’ before ‘<’ token
 typedef</span> <span class="pl-k">struct</span> _XfceHVBox      XfceHVBox;</td>
                                                                           ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:620:70: error: stray ‘#’ in program
        <td id="LC38" class="blob-code blob-code-inner js-file-line">#<span class="pl-k">define</span> <span class="pl-en">XFCE_TYPE_HVBOX</span>            (xfce_hvbox_get_type())</td>
                                                                     ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:624:70: error: stray ‘#’ in program
        <td id="LC39" class="blob-code blob-code-inner js-file-line">#<span class="pl-k">define</span> <span class="pl-en">XFCE_HVBOX</span>(<span class="pl-v">obj</span>)            (G_TYPE_CHECK_INSTANCE_CAST (obj, XFCE_TYPE_HVBOX, XfceHVBox))</td>
                                                                     ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:628:70: error: stray ‘#’ in program
        <td id="LC40" class="blob-code blob-code-inner js-file-line">#<span class="pl-k">define</span> <span class="pl-en">XFCE_HVBOX_CLASS</span>(<span class="pl-v">klass</span>)    (G_TYPE_CHECK_CLASS_CAST (klass, XFCE_TYPE_HVBOX, XfceHVBoxClass))</td>
                                                                     ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:632:70: error: stray ‘#’ in program
        <td id="LC41" class="blob-code blob-code-inner js-file-line">#<span class="pl-k">define</span> <span class="pl-en">XFCE_IS_HVBOX</span>(<span class="pl-v">obj</span>)         (G_TYPE_CHECK_INSTANCE_TYPE (obj, XFCE_TYPE_HVBOX))</td>
                                                                     ^
In file included from /usr/include/glib-2.0/gobject/gobject.h:24:0,
                 from /usr/include/glib-2.0/gobject/gbinding.h:29,
                 from /usr/include/glib-2.0/glib-object.h:23,
                 from /usr/include/glib-2.0/gio/gioenums.h:28,
                 from /usr/include/glib-2.0/gio/giotypes.h:28,
                 from /usr/include/glib-2.0/gio/gio.h:26,
                 from /usr/include/gtk-2.0/gdk/gdkapplaunchcontext.h:30,
                 from /usr/include/gtk-2.0/gdk/gdk.h:32,
                 from /usr/include/gtk-2.0/gtk/gtk.h:32,
                 from wck-plugin.c:26:
/usr/include/glib-2.0/gobject/gtype.h:2229:2: error: expected identifier or ‘(’ before ‘)’ token
 }))
  ^
/usr/include/glib-2.0/gobject/gtype.h:491:66: note: in expansion of macro ‘_G_TYPE_CIT’
 #define G_TYPE_CHECK_INSTANCE_TYPE(instance, g_type)            (_G_TYPE_CIT ((instance), (g_type)))
                                                                  ^~~~~~~~~~~
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:632:185: note: in expansion of macro ‘G_TYPE_CHECK_INSTANCE_TYPE’
 pl-en">XFCE_IS_HVBOX</span>(<span class="pl-v">obj</span>)         (G_TYPE_CHECK_INSTANCE_TYPE (obj, XFCE_TYPE_HVBOX))</td>
                                                                     ^~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/libxfce4panel.h:33:0,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-plugin-provider.h:23,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-macros-46.h:26,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-plugin.h:28,
                 from wck-plugin.h:25,
                 from wck-plugin.c:29:
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:636:70: error: stray ‘#’ in program
        <td id="LC42" class="blob-code blob-code-inner js-file-line">#<span class="pl-k">define</span> <span class="pl-en">XFCE_IS_HVBOX_CLASS</span>(<span class="pl-v">klass</span>) (G_TYPE_CHECK_CLASS_TYPE ((klass), XFCE_TYPE_HVBOX))</td>
                                                                     ^
In file included from /usr/include/glib-2.0/gobject/gobject.h:24:0,
                 from /usr/include/glib-2.0/gobject/gbinding.h:29,
                 from /usr/include/glib-2.0/glib-object.h:23,
                 from /usr/include/glib-2.0/gio/gioenums.h:28,
                 from /usr/include/glib-2.0/gio/giotypes.h:28,
                 from /usr/include/glib-2.0/gio/gio.h:26,
                 from /usr/include/gtk-2.0/gdk/gdkapplaunchcontext.h:30,
                 from /usr/include/gtk-2.0/gdk/gdk.h:32,
                 from /usr/include/gtk-2.0/gtk/gtk.h:32,
                 from wck-plugin.c:26:
/usr/include/glib-2.0/gobject/gtype.h:2239:2: error: expected identifier or ‘(’ before ‘)’ token
 }))
  ^
/usr/include/glib-2.0/gobject/gtype.h:559:66: note: in expansion of macro ‘_G_TYPE_CCT’
 #define G_TYPE_CHECK_CLASS_TYPE(g_class, g_type)                (_G_TYPE_CCT ((g_class), (g_type)))
                                                                  ^~~~~~~~~~~
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:636:185: note: in expansion of macro ‘G_TYPE_CHECK_CLASS_TYPE’
 pl-en">XFCE_IS_HVBOX_CLASS</span>(<span class="pl-v">klass</span>) (G_TYPE_CHECK_CLASS_TYPE ((klass), XFCE_TYPE_HVBOX))</td>
                                                                     ^~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/libxfce4panel.h:33:0,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-plugin-provider.h:23,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-macros-46.h:26,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-plugin.h:28,
                 from wck-plugin.h:25,
                 from wck-plugin.c:29:
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:640:70: error: stray ‘#’ in program
        <td id="LC43" class="blob-code blob-code-inner js-file-line">#<span class="pl-k">define</span> <span class="pl-en">XFCE_HVBOX_GET_CLASS</span>(<span class="pl-v">obj</span>)  (G_TYPE_INSTANCE_GET_CLASS ((obj), XFCE_TYPE_HVBOX, XfceHVBoxClass))</td>
                                                                     ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:665:72: error: expected identifier or ‘(’ before ‘<’ token
         <td id="LC49" class="blob-code blob-code-inner js-file-line">};</td>
                                                                        ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:723:72: error: expected identifier or ‘(’ before ‘<’ token
         <td id="LC63" class="blob-code blob-code-inner js-file-line">};</td>
                                                                        ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:732:187: error: expected identifier or ‘(’ before ‘<’ token
 box_get_type</span>         (<span class="pl-k">void</span>) G_GNUC_CONST;</td>
                                                                           ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:749:180: error: expected identifier or ‘(’ before ‘<’ token
          gint            spacing) G_GNUC_MALLOC G_GNUC_WARN_UNUSED_RESULT;</td>
                                                                           ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:762:144: error: expected identifier or ‘(’ before ‘<’ token
                                              GtkOrientation  orientation);</td>
                                                                           ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:766:165: error: expected identifier or ‘(’ before ‘<’ token
  class="pl-en">xfce_hvbox_get_orientation</span>  (XfceHVBox      *hvbox);</td>
                                                                           ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:784:70: error: stray ‘#’ in program
        <td id="LC77" class="blob-code blob-code-inner js-file-line">#<span class="pl-k">endif</span> <span class="pl-c"><span class="pl-c">/*</span> !__XFCE_HVBOX_H__ <span class="pl-c">*/</span></span></td>
                                                                     ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:808:10: warning: missing terminating ' character
     <!-- '"` --><!-- </textarea></xmp> --></option></form><form class="js-jump-to-line-form" action="" accept-charset="UTF-8" method="get"><input name="utf8" type="hidden" value="&#x2713;" />
          ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:808:10: error: missing terminating ' character
     <!-- '"` --><!-- </textarea></xmp> --></option></form><form class="js-jump-to-line-form" action="" accept-charset="UTF-8" method="get"><input name="utf8" type="hidden" value="&#x2713;" />
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:827:31: error: expected identifier or ‘(’ before numeric constant
       <li class="mr-3">&copy; 2018 <span title="0.23176s from unicorn-919311818-z1vpl">GitHub</span>, Inc.</li>
                               ^~~~
In file included from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/libxfce4panel.h:33:0,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-plugin-provider.h:23,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-macros-46.h:26,
                 from /usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-panel-plugin.h:28,
                 from wck-plugin.h:25,
                 from wck-plugin.c:29:
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:860:12: warning: missing terminating ' character
     You can't perform that action at this time.
            ^
/usr/include/xfce4/libxfce4panel-2.0/libxfce4panel/xfce-hvbox.h:860:12: error: missing terminating ' character
     You can't perform that action at this time.
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Makefile:515: fallo en las instrucciones para el objetivo 'libwck_common_la-wck-plugin.lo'
make[2]: *** [libwck_common_la-wck-plugin.lo] Error 1
make[2]: se sale del directorio '/home/pablo/Descargas/wd/xfce4-windowck-plugin-master/common'
Makefile:479: fallo en las instrucciones para el objetivo 'all-recursive'
make[1]: *** [all-recursive] Error 1
make[1]: se sale del directorio '/home/pablo/Descargas/wd/xfce4-windowck-plugin-master'
Makefile:409: fallo en las instrucciones para el objetivo 'all'
make: *** [all] Error 2
```


> Hi, @arodu and @phw you must download libxfce4panel/xfce-hvbox.h source file and store it to expected location. It can be found on github somewhere. I believe this was removed from xfce-panel sources because of GTK2->GTK3 migration. But once build GTK2 panel plugin works without problems.

**Comment by shmutalov** [comment]
Packaged under Debian Sid (amd64):

https://drive.google.com/open?id=1ylqd9sjL9aBt_1a5jkVbcANgsdJLBNu6
